pytensor/tensor/basic.py

```diff
@@ -3207,13 +3207,14 @@ def tile(
     return A_replicated.reshape(tiled_shape)
 
 
-class ARange(Op):
+class ARange(COp):
     """Create an array containing evenly spaced values within a given interval.
 
     Parameters and behaviour are the same as numpy.arange().
 
     """
 
+    # TODO: Arange should work with scalars as inputs, not arrays
```

Comment:
This would just require some type checking in the perform method?

Reply:
make_node as well